package.json

```diff
@@ -1,18 +1,29 @@
 {
   "name": "github-dark",
+  "title": "GitHub Dark",
   "version": "1.22.117",
   "description": "Dark GitHub style",
   "license": "CC-BY-SA-4.0",
   "repository": "StylishThemes/GitHub-Dark",
   "homepage": "https://github.com/StylishThemes/GitHub-Dark",
+  "main": "github-dark.css",
   "engines": {
     "node": ">=12"
   },
   "scripts": {
-    "clean": "grunt clean",
-    "generate": "grunt generate",
-    "test": "grunt lint",
-    "usercss": "grunt usercss"
+    "authors": "bash tools/authors.sh",
+    "clean": "npm run perfectionist && node tools/fix-perfectionist.js",
+    "generate": "node tools/generate.js",
+    "eslint": "npx eslint --quiet --color *.js tools/*.js",
```

Comment:
Using `npx` isn't needed here. By default Node loads these from the local `node_modules` folder. All this is doing is making the tasks take longer to execute because it's downloading and installing the packages before running them.

Reply:
Actually `npx` should too take from the local folder, but I usually observe it downloading anways, so not sure. Generally `npx` is a rather unmaintained tool and should be avoided, like it's possible in these scripts here, so go for it.

Reply:
npx replace with?

Reply:
No need to replace, just remove. https://github.com/xt0rted/eslint-action/blob/f65f062bbb357a5ba9eb67510d0a6cdbd981c71a/package.json#L5-L9

Reply:
Feel free to push any chages to this branch, I dont have that action in this repo and Ive no idea about removing npx.

Reply:
@xt0rted you do realize theres no such action i this repo? How can I remove it here? so confusing.

Ill leave it as is so not to do wrong thing, if anyone want to make alterations please push them into this branch.

Reply:
This is what we're saying to change them to:

```json
"scripts": {
  "authors": "bash tools/authors.sh",
  "clean": "npm run perfectionist && node tools/fix-perfectionist.js",
  "generate": "node tools/generate.js",
  "eslint": "eslint --quiet --color *.js tools/*.js",
  "stylelint": "stylelint github-dark.css themes/src/**/*.css",
  "imagemin": "bash tools/imagemin.sh",
  "test": "npm run eslint && npm run stylelint",
  "usercss": "node tools/build-usercss.js",
  "perfectionist": "perfectionist github-dark.css github-dark.css --indentSize 2 --maxAtRuleLength 250",
  "patch": "ver -p -d patch github-dark.css github-dark.user.css",
  "minor": "ver -p -d minor github-dark.css github-dark.user.css",
  "major": "ver -p -d major github-dark.css github-dark.user.css",
  "update": "updates -cu && npm install"
},
```